lib/resources/BitcoinReceivers.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'bitcoin/receivers',
+
+  includeBasic: [
+    'create', 'list', 'retrieve'
+  ],
+
+  listBitcoinTransactions: stripeMethod({
+    method: 'GET',
+    path: '/{receiverId}/transactions',
```

Comment:
Do BitcoinTransactions not need their own resource? Can't retrieve by ID/update?